### DISCUSSION TITLE
Splitter 控件折叠，再展开，宽度不是折叠前时的大小

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#52003] [General]
<Splitter.Panel collapsible min="20%" defaultSize={200} max={'80%'}>
{children}
</Splitter.Panel>

如上面的设置， 默认打开时， 显示的宽度是 200PX， 这个时候点一下折叠， 再展开， 显示的宽度就变成了 20%, 如果人为拖动改变了大小， 折叠再展开， 宽度也变成了 min 这个属性的值了

**Comment by Wxh16144** [comment]
和 https://github.com/ant-design/ant-design/issues/51152#issuecomment-2398729975 这个是同一个问题么？ 我没有用过这个组件，但是看你的问题标题很眼熟

**Reply by ekuku1984** [reply]
谢谢！！
是同一个问题
